pkg/apis/core/annotation_key_constants.go

```diff
@@ -23,6 +23,10 @@ const (
 	// webhook backend fails.
 	ImagePolicyFailedOpenKey string = "alpha.image-policy.k8s.io/failed-open"
 
+	// ImagePolicyAuditRequired is recommended for image review backends to
+	// add when an pod is allowed to be created, but should be audited.
+	ImagePolicyAuditRequired string = "alpha.image-policy.k8s.io/audit-required"
```

Comment:
This sounds like something that would have something to do with Kubernetes audit logging, but it just means a human should look at it (IIUC). I think it should be renamed. Alternatively, if this is opaque to Kubernetes, it doesn't actually need to be a part of the k8s API (preferred).

Reply:
its not clear why the `ImagePolicyFailedOpenKey` also had to be in the public API.  Its only used in the admission plugin from what I can gather, why can the keys not be kept local to the plugin?

Reply:
or why this is not in the /pkg/apis/imagepolicy... instead.

Reply:
These may be used by either humans in audits, or by automated systems validating the pods at a later time (e.g., Kritis or other auditing products related to Binary Authorization).

I have no strong opinion about the placement, I added the new constant to the pre-existing one.   Happy to move them somewhere under imagepolicy.

Reply:
> its not clear why the ImagePolicyFailedOpenKey also had to be in the public API

It was added before we had a notion of "audit annotations", as a hack of getting the information into the audit logs. Although I agree it didn't really need to be a part of the official API.